Make `encounter_deletion` general for low and increased risk encounters

Ein-Tim opened the pull request:
This PR updates the text of the FAQ entry `encounter_deletion`. It makes the text general, so that the FAQ is not specific to low risk encounters anymore.

<details>
<summary>Preview of the change</summary>

| German | English | 
|---|---|
|![DE](https://user-images.githubusercontent.com/67682506/151957181-8aeee869-5072-498e-afb8-4b51b76b20fb.png)| ![EN](https://user-images.githubusercontent.com/67682506/151957187-4571f8d8-dcbd-432f-9093-7d1deeb3e3f4.png)|



</details>

MikeMcC399 commented:
@Ein-Tim 

What about the fact that the date of the encounter is shown in the Journal?  Probably this article was written before that enhancement was introduced in 1.12 (https://www.coronawarn.app/en/blog/2021-02-10-corona-warn-app-version-1-12/).

This point could be added to the FAQ so users are aware that they can look in the Journal before the exposure is deleted.

Ein-Tim commented:
@MikeMcC399 

This sounds like a good idea and I will add a commit implementing this soon.

Ein-Tim commented:
@MikeMcC399 

I added a reference to the blog post with [`2baf401`](https://github.com/corona-warn-app/cwa-website/pull/2394/commits/2baf401e7fd672cdffb1d86a79d42f09d321f077).
I'd be happy to hear your feedback!

Ein-Tim commented on src/data/faq_de.json:
```diff
@@ -1902,11 +1902,12 @@
                         "textblock": [
-                            "Sobald die App die aktuellen Diagnoseschlüssel (Positivkennungen) vom Server lädt, wird eine Überprüfung des Kontaktprotokolls durchgeführt. Dabei wird für jeden der letzten 14 Tage, für den Diagnoseschlüssel vorhanden sind, eine separate Überprüfung des Kontaktprotokolls durchgeführt. Das bedeutet also, die App gleicht die Begegnungen der letzten 14 Tage mit den Posi ...
+                            "Sobald die App die aktuellen Diagnoseschlüssel (Positivkennungen) vom Server lädt, wird eine Überprüfung des Kontaktprotokolls durchgeführt. Dabei wird für jeden der letzten 14 Tage, für den Diagnoseschlüssel vorhanden sind, eine separate Überprüfung des Kontaktprotokolls durchgeführt. Das bedeutet also, die App gleicht die Begegnungen der letzten 14 Tage mit den Posi ...
+                            "Sie finden mehr Informationen dazu, wann die Risiko-Begegnung stattgefunden hat im Kontakt-Tagebuch. Weitere Informationen dau finden Sie im Blog Post <a href='../blog/2021-02-10-corona-warn-app-version-1-12' target='_blank'>Corona-Warn-App Version 1.12 mit zwei Neuerungen</a> im Abschnitt 'Kontakt-Tagebuch mit Begegnungshistorie'."
```
Typos.
```suggestion
                            "Sie finden mehr Informationen dazu, wann die Risiko-Begegnung stattgefunden hat, im Kontakt-Tagebuch. Weitere Informationen zu dieser Funktion finden Sie im Blog Post <a href='../blog/2021-02-10-corona-warn-app-version-1-12' target='_blank'>Corona-Warn-App Version 1.12 mit zwei Neuerungen</a> im Abschnitt 'Kontakt-Tagebuch mit Begegnungshistorie'."
```

Ein-Tim commented on src/data/faq.json:
```diff
@@ -1894,11 +1894,12 @@
                         "textblock": [
-                            "As soon as the app downloads the current diagnostic keys (positive IDs) from the server, a check of the contact log is performed. For each of the last 14 days for which diagnostic keys are available, a separate contact log check is performed. This means that the app compares the encounters of the last 14 days with the positive IDs to check whether there has been a con ...
+                            "As soon as the app downloads the current diagnosis keys (positive IDs) from the server, a check of the contact log is performed. For each of the last 14 days for which diagnosis keys are available, a separate contact log check is performed. This means that the app compares the encounters of the last 14 days with the positive IDs to check whether there has been a conta ...
+                            "You can find more information on when your risk encounter happened in the contact journal. More information on this are available in the blog post <a href='../blog/2021-02-10-corona-warn-app-version-1-12' target='_blank'>Corona-Warn-App version 1.12 comes with two new features</a> in the paragraph 'Contact journal with exposure history'."
```
Rewording.

```suggestion
                            "You can find more information on when your risk encounter happened in the contact journal. Further information on this feature are available in the blog post <a href='../blog/2021-02-10-corona-warn-app-version-1-12' target='_blank'>Corona-Warn-App version 1.12 comes with two new features</a> in the paragraph 'Contact journal with exposure history'."
```

Ein-Tim reviewed (commented):
Spotted two places which need improvement.

MikeMcC399 commented on src/data/faq.json:
```diff
@@ -1904,11 +1904,12 @@
                         "textblock": [
-                            "As soon as the app downloads the current diagnostic keys (positive IDs) from the server, a check of the contact log is performed. For each of the last 14 days for which diagnostic keys are available, a separate contact log check is performed. This means that the app compares the encounters of the last 14 days with the positive IDs to check whether there has been a con ...
+                            "As soon as the app downloads the current diagnosis keys (positive IDs) from the server, a check of the contact log is performed. For each of the last 14 days for which diagnosis keys are available, a separate contact log check is performed. This means that the app compares the encounters of the last 14 days with the positive IDs to check whether there has been a conta ...
+                            "You can find more information on when your risk encounter happened in the contact journal. Further information on this feature are available in the blog post <a href='../blog/2021-02-10-corona-warn-app-version-1-12' target='_blank'>Corona-Warn-App version 1.12 comes with two new features</a> in the paragraph 'Contact journal with exposure history'."
```
"are" => "is" 
in
"Further information on this feature are ..."

"information" is an uncountable noun always used in the singular, see https://dictionary.cambridge.org/grammar/british-grammar/information

Ein-Tim replied:
```suggestion
                            "You can find more information on when your risk encounter happened in the contact journal. Further information on this feature is available in the blog post <a href='../blog/2021-02-10-corona-warn-app-version-1-12' target='_blank'>Corona-Warn-App version 1.12 comes with two new features</a> in the paragraph 'Contact journal with exposure history'."
```

MikeMcC399 reviewed (changes requested):
One grammar point "information ... is" to correct.

Ein-Tim commented:
@dsarkar 

Will you have time to look at this soon?

MikeMcC399 commented:
@Ein-Tim 
You currently have conflicts. It would be helpful to resolve these before further reviewing.

Ein-Tim commented:
@MikeMcC399 

Thanks! Resolved merge conflicts in [`fc3a513`](https://github.com/corona-warn-app/cwa-website/pull/2394/commits/fc3a51375ca85390ee780afa2dc861fc53dbf4fd).

MikeMcC399 commented:
@Ein-Tim 

Unfortunately I now think that my suggestion to mention the contact journal was a  bad one!

In the case of `#encounter_deletion` the situation is that a risk encounter is already deleted, so there will be nothing from the deleted risk encounter to see in the contact journal. The risk encounter will also have disappeared in the contact journal. If there have been multiple encounters, then there may be other ones listed in the contact journal, but **not the one which has disappeared** (by definition!).

I'm very sorry, but I think this paragraph should be removed. What do you think?

dsarkar commented:
Morning, @Ein-Tim @MikeMcC399! We will have a look tomorrow! Enjoy the weekend!

Ein-Tim commented:
@MikeMcC399 

Yes, you're right! No need to apologize, I also didn't think about this! Will remove the paragraph tomorrow. 

Thank you @dsarkar!

Ein-Tim commented:
[`c9a84fe`](https://github.com/corona-warn-app/cwa-website/pull/2394/commits/c9a84fe31d07e02011f35a351deb2427f4588a5e) removed the reference to the blog post. I rebased to the current master, this PR is ready to be merged now.

Ein-Tim commented:
I rebased on the current master, squashed all commits and removed the reference to the blog post in [`1aed72c`](https://github.com/corona-warn-app/cwa-website/pull/2394/commits/1aed72ceed5f465f91bb21eec52a85889bf8e53d).

Ready to be merged!

Ein-Tim commented:
Oh no, I just see that there are many unwanted commits in here, stand by please...

Ein-Tim commented:
Continue in #2421.
